meshroom/ui/graph.py

```diff
@@ -952,6 +952,20 @@ def selectFollowing(self, node):
         """ Select all the nodes the depend on 'node'. """
         self.selectNodes(self._graph.dfsOnDiscover(startNodes=[node], reverse=True, dependenciesOnly=True)[0])
 
+    @Slot(str)
+    def setSelectedNodesColor(self, color):
+        """ Sets the Provided color on the selected Nodes.
+
+        Args:
+            color (str): Hex code of the color to be set on the nodes.
+        """
+        # Update the color attribute of the nodes which are currently selected
+        with self.groupedGraphModification("Update Color for Select Nodes"):
```

Comment:
For the command name that appears in the undo stack:

```suggestion
        with self.groupedGraphModification("Set Nodes Color"):
```

I think we could go with removing the notion of "selected" in those, as selection operations are not undoable - therefore, undoing "Set Selected Nodes Color" is a bit ambiguous if the selection has changed in between.
What do you think?

Reply:
Yes, makes sense, I probably did not notice this while working on the previous comments. Thanks for the suggestion.